py-polars/polars/internals/dataframe/frame.py

```diff
@@ -4822,12 +4822,17 @@ def drop_in_place(self, name: str) -> pli.Series:
         """
         return pli.wrap_s(self._df.drop_in_place(name))
 
-    def cleared(self) -> Self:
+    def cleared(self, n: int = 0) -> Self:
```

Comment:
I think we should call this `clear`. We do so with other methods:

`sort -> sorted`
`slice -> sliced`

Reply:
Agreed; and done... ;)